package.json

```diff
@@ -23,9 +23,14 @@
 		"readme.txt",
 		"wp-feature-notifications.php"
 	],
+	"workspaces": [
+		"./storybook"
+	],
 	"scripts": {
 		"start": "wp-scripts start",
+		"start:storybook": "npm run start --workspace=storybook",
 		"build": "wp-scripts build",
+		"build:storybook": "npm run build --workspace=storybook",
 		"prephp-install": "wp-env start",
```

Comment:
i've noticed also there two that are "duplicated", is worth to remove them in this PR? If yes do it, however check that they are not used in the php test workflow

Reply:
@erikyo which two are duplicated?